test/e2e/storage/drivers/in_tree.go

```diff
@@ -1305,11 +1313,13 @@ func InitVSphereDriver() testsuites.TestDriver {
 				"", // Default fsType
 				"ext4",
 			),
+			TopologyKeys: []string{v1.LabelZoneFailureDomain},
```

Comment:
zones in vSphere are optional, IMO we can't assume that they support topology by default.

https://github.com/kubernetes/kubernetes/blob/a36704b444518f1736f5f24197d007e642ffb66a/staging/src/k8s.io/legacy-cloud-providers/vsphere/vsphere.go#L1591-L1595

Reply:
I removed the Keys.